src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -45,7 +45,7 @@ public class OtlpExporter : ActivityExporter
         public OtlpExporter(OtlpExporterOptions options)
         {
             this.headers = options.Headers;
-            this.channel = new Channel(options.Endpoint, options.Credentials);
+            this.channel = new Channel(options.Endpoint, options.Credentials, options.ChannelOptions);
```

Comment:
null is totally fine here:
see https://github.com/grpc/grpc/blob/7faa7795046090ee5fc2ac86b9892c985950307b/src/csharp/Grpc.Core/Channel.cs#L55